beacon-chain/p2p/broadcaster.go

```diff
@@ -228,6 +229,57 @@ func (s *Service) BroadcastBlob(ctx context.Context, subnet uint64, blob *ethpb.
 	return nil
 }
 
+func (s *Service) BroadcastBlockChunks(ctx context.Context, cBlk *ethpb.ChunkedBeaconBlock) error {
+	ctx, span := trace.StartSpan(ctx, "p2p.BroadcastBlob")
+	defer span.End()
+	if cBlk == nil {
+		return errors.New("attempted to broadcast nil blob sidecar")
+	}
+	forkDigest, err := s.currentForkDigest()
+	if err != nil {
+		err := errors.Wrap(err, "could not retrieve fork digest")
+		tracing.AnnotateError(span, err)
+		return err
+	}
+	topic, ok := GossipTypeMapping[reflect.TypeOf(&ethpb.BeaconBlockChunk{})]
```

Comment:
No need to use reflect when you already know the topic and it's exported

```suggestion
	topic := RLNCTopicFormat
```